apps/dashboard/src/@/constants/thirdweb.server.ts

```diff
@@ -24,8 +31,43 @@ export function getThirdwebClient(jwt?: string) {
       pay: THIRDWEB_PAY_DOMAIN,
       storage: THIRDWEB_STORAGE_DOMAIN,
       social: THIRDWEB_SOCIAL_API_DOMAIN,
+      bundler: THIRDWEB_BUNDLER_DOMAIN,
     });
   }
+
+  if (!getTransactionDecorator()) {
+    setTransactionDecorator(async ({ account, transaction }) => {
```

Comment:
@jnsdls @gregfromstl making sure you both see this - introduced a 'transaction hook' that lets you 'decorate' accounts and transactions globally. Feels like its gonna be very useful in the future, and works great for this use case where i need to convert / add paymaster data to all transactions on this particular chain